### DISCUSSION TITLE
web components 组件可以接入 taro 使用么？

### DISCUSSION THREAD

**Discussion by Howie126313** [NervJS/taro#13667] [Q & A] [answered]
### 这个特性解决了什么问题？
现阶段有一个 vue3 的 h5 项目，其中 UI 组件库是使用 web components 封装。现在想要同构小程序，如果支持的话可以降低开发成本。

### 这个 API 长什么样？
taro 开发 web component 的挂载的方法

**Comment by Chen-jj** [comment, marked as answer]
Taro 在小程序端还没有模拟 WebComponents 的相关 API，因此还不支持

**Comment by Howie126313** [comment]
@Chen-jj thx~
